Engine/source/gui/3d/guiTSControl.cpp

```diff
@@ -578,14 +578,12 @@ void GuiTSCtrl::onRender(Point2I offset, const RectI &updateRect)
       GFXTextureObject *texObject = mStereoGuiTarget->getTexture(0);
       const FovPort *currentFovPort = GFX->getStereoFovPort();
       const MatrixF *eyeTransforms = GFX->getStereoEyeTransforms();
-      const MatrixF *worldEyeTransforms = GFX->getInverseStereoEyeTransforms();
       const Point3F *eyeOffset = GFX->getStereoEyeOffsets();
 
       for (U32 i=0; i<2; i++)
       {
          GFX->activateStereoTarget(i);
          Frustum gfxFrustum = originalFrustum;
-         const F32 frustumDepth = gfxFrustum.getNearDist();
```

Comment:
Somewhat distressing that it's a overridden local var too.

If we're not changing the frustum in there at all, then I'd agree, we should just be able to push the frustumDepth declaration to just before the loop, and then use it again on line 612 and rename THAT var to like, furstumDepthAdjusted or something.

Seem reasonable?

Other than that, I didn't run into any issues with this. So if we get that part tweaked we should be good to roll it in.